modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -120,7 +120,7 @@ enum class Feature(
         title = "Cache entire playing episode",
         defaultValue = BuildConfig.DEBUG,
         tier = FeatureTier.Free,
-        hasFirebaseRemoteFlag = false,
+        hasFirebaseRemoteFlag = true,
```

Comment:
The feature depends on `exoplayer_cache_entire_playing_episode_size_in_mb` value in Firebase remote config. It is currently set to 0 and disabled for the release build.